### PR TITLE
Fix #33: Avoid redirection on the homepage

### LINKED ISSUE #33: Avoid redirection on the homepage
Currently, we have a redirect on the root page, which is a bit annoying. We can improve the index page and navbar include by:

- Adding an optional page variable for context path, i.e. `contextPath: docs/2020.05/`.
- Incorporate this new variable to `index.html` and `navbar.html` templates to change the links to the latest versions.
- Replacing the template for the main index page (`redirect` => `index`) and using the new variable (only in the root index file).
- Regenerating the docs.

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

**Comment by petrdvorak** [comment]
- I don't see any broken links, please be more specific.
- The obvious issue is that the top file uses HTML redirection. Until we can do HTTP 302 redirection, this will always result in a crappy experience on page loads (user looking at some redirect page) and probably some SEO penalties as well.
- We will change the structure of the documentation anyway, to support versioning in a smarter way. We already can see that the "one page per big release" has limits (different components develop in different pace).
- The Algolia search engine supports "versioning" and to a large extent can handle such situations.
- Google is better off if we have an actual page on domain root.

**Comment by hvge** [comment]
Just test your own change. The new homepage has empty selected tutorials section:
Screen Shot 2020-06-08 at 12 56 37

**Comment by petrdvorak** [comment]
Probably an issue on your end, I can see those sections just fine. Did you update the documentation for current and develop?

**Comment by hvge** [comment]
Still broken :D

File: app/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-layout: redirect
-sitemap: false
-redirect_to: /docs/2020.05
----
\ No newline at end of file
+layout: index
+contextPath: docs/2020.05/
```
Comment (hvge) [inline app/index.html:3]: I think this has to start with backslash. LIke `/docs/2020.05`

File: scripts/bundler.sh [hunk 2]
```diff
@@ -251,11 +251,11 @@ function DO_NEW
...
 	local ROOT_INDEX_FILE="${APP_DIR}/index.html"
-	echo "---" 									>  "$ROOT_INDEX_FILE"
-	echo "layout: redirect"						>> "$ROOT_INDEX_FILE"
-	echo "sitemap: false"						>> "$ROOT_INDEX_FILE"
-	echo "redirect_to: /docs/${NEW_RELEASE}"	>> "$ROOT_INDEX_FILE"
-	echo -n "---" 								>> "$ROOT_INDEX_FILE"
+	echo "---" 								>  "$ROOT_INDEX_FILE"
+	echo "layout: index"					>> "$ROOT_INDEX_FILE"
+	echo "releaseIdentifier: $NEW_RELEASE"	>> "$ROOT_INDEX_FILE"
+	echo "contextPath: docs/$NEW_RELEASE/"	>> "$ROOT_INDEX_FILE"
```
Comment (hvge) [inline scripts/bundler.sh:257]: The same here. The local path should start with backslash.

**Pull request merged**